File: 44-integrations/npm-package/integrations.ts
import axios from 'axios';
const API = 'https://cost-katana-backend.store/api'\;
const KEY = process.env.COST_KATANA_API_KEY;

export async function listIntegrations() {
  const res = await axios.get(`${API}/integrations`,
    {headers: {'Authorization': `Bearer ${KEY}`}});
  console.log('🔌 Integrations:', res.data.data.length);
  return res.data.data;
}

export async function connectIntegration(provider: string, credentials: any) {
  const res = await axios.post(`${API}/integrations/connect`, {provider, credentials},
    {headers: {'Authorization': `Bearer ${KEY}`}});
  console.log('✅ Integration connected:', provider);
  return res.data.data;
}

File: 44-integrations/python-sdk/integrations.py
import os, requests
API = "https://cost-katana-backend.store/api"
KEY = os.getenv("COST_KATANA_API_KEY")

def list_integrations():
    res = requests.get(f"{API}/integrations",
        headers={"Authorization": f"Bearer {KEY}"})
    res.raise_for_status()
    print(f"🔌 Integrations: {len(res.json()['data'])}")
    return res.json()["data"]

def connect_integration(provider, credentials):
    res = requests.post(f"{API}/integrations/connect",
        json={"provider": provider, "credentials": credentials},
        headers={"Authorization": f"Bearer {KEY}"})
    res.raise_for_status()
    print(f"✅ Integration connected: {provider}")
    return res.json()["data"]
